src/engine/source/README.md

```diff
@@ -0,0 +1,77 @@
+# Source
+
+1. [Directory structure](#directory-structure)
+2. [Component structure](#component-structure)
+
+# Directory structure
+
+```plaintext
+source/
+│
+├── api/
+├── base/
+├── bk/
+├── builder/
```

Comment:
missing indexerconnector, vdscanner, feedmanager and apiserver, is it really necessary to have this? I don't think we will add many modules, but it is something easy to become obsolete and forget.